script/generate-ebook

```diff
@@ -0,0 +1,42 @@
+#/bin/env bash
```

Comment:
`#!/bin/bash` or `#!/usr/bin/env bash` would be better here. Would also be good to add a `set -e` so errors aren't silently ignored.

Reply:
https://guides.github.com/features/mastering-markdown/

Reply:
I'd even suggest:
```bash
#! /usr/bin/env bash

if [ -n "$DEBUG" ]
then
  set -x
fi
set -euo pipefail
# ref: https://coderwall.com/p/fkfaqq/safer-bash-scripts-with-set-euxo-pipefail
```